lib/config.js

```diff
@@ -1,7 +1,7 @@
 var util = require('util');
 var defaults = require('lodash.defaults');
 
-var ConfigError = function ConfigError(message, extra) {
+var ConfigError = function (message, extra) {
```

Comment:
Well it seems useless, but the function name was there for a reason ;)
Without the name the stucktrace does not label the error correctly.